### PR TITLE
rename/move api.md to cli.md

### PR DESCRIPTION
**Pull request by nplanel**

### CODE REVIEW

**Comment by nplanel** [comment]
aim to fix documentation sentence like 
"Skydive CLI, see http://skydive.network/documentation/cli"

File: documentation/cli.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Documentation
-section: API
+section: CLI
```
Comment (safchain) [inline documentation/cli.md]: Will be better to write "Command line interface" as it will be the title on the page.

Comment (nplanel) [reply documentation/cli.md]: done

File: _layouts/doc.html [hunk 2]
```diff
@@ -83,7 +83,7 @@ <h4><a href="/documentation/contribute">Contribute</a></h4>
         </div>
         <div class="row widget">
           <div class="col-xs-12">
-            <h4><a href="/documentation/api">Command line interface</a></h4>
+            <h4><a href="/documentation/cli">Command line interface</a></h4>
             <ul>
               <li><a href="/documentation/api#topology-requests">Topology requests</a></li>
```
Comment (safchain) [inline _layouts/doc.html]: s/api/cli/

**Pull request merged**